### DISCUSSION TITLE
Redirect back to existing tab after sign up

### DISCUSSION THREAD

**Discussion by bennik88** [supabase/supabase#9040] [Questions]
Does anyone know how to implement a redirection back to the already open sign up page tab when clicking the verification mail link after sign up?
I experimented with the Supabase E-Mail templates as well as with the `options: {redirect:...` without success.

**Comment by saltcod** [comment]
Hey @bennik88 — you can redirect to a page after login, as you've likely discovered already. I don't think you can point to a tab that's already open though.
